lib/user_fiendly_formatter.ex

```diff
@@ -34,17 +34,21 @@ defmodule PrettyLog.UserFriendlyFormatter do
       |> String.upcase()
       |> String.pad_trailing(5)
 
-    padded_message =
-      message
-      |> TextSanitizer.sanitize()
-      |> String.pad_trailing(48)
+    sanitized_message = TextSanitizer.sanitize(message)
 
     encoded_metadata =
       (pre_meta ++ metadata)
       |> TextSanitizer.sanitize_keyword()
       |> Logfmt.encode()
 
-    "#{time_string}\t|#{level_string}| #{padded_message}\t| #{encoded_metadata}\n"
+    encoded_metadata =
+      if encoded_metadata != "" do
+        "  " <> encoded_metadata
+      else
+        encoded_metadata
+      end
+
+    "#{time_string}\t|#{level_string}| #{sanitized_message}#{encoded_metadata}\n"
```

Comment:
Since `encoded_metadata` is empty in the else branch, I think it's clearer just doing
```elixir
      if encoded_metadata != "" do
           "#{time_string}\t|#{level_string}| #{sanitized_message} #{encoded_metadata}\n"
      else
           "#{time_string}\t|#{level_string}| #{sanitized_message}\n"
      end
```